src/app/referee/referee.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="referee-container">
-	<div class="upper-container" [ngClass]="{stopCommand: Command[refereeMessage.command] == 'STOP', haltCommand: Command[refereeMessage.command] == 'HALT'}">
+	<div class="upper-container" [ngClass]="{stopCommand: Command[refereeMessage.command] == 'STOP' || Command[refereeMessage.command].substring(0, 14) == 'BALL_PLACEMENT', haltCommand: Command[refereeMessage.command] == 'HALT'}">
```

Comment:
can you move all the checks into methods and call those?

Reply:
yes, I will add the stop status to timeouts as well